excluding already sent qsl cards and ignored

dg9vh opened the pull request:
the open_qso_list should only show QSO-data that has not been confirmed outgoing. Also ignored cards should not be offered to be added to print list.

AndreasK79 commented:
@dg9vh I agree with not displaying ignored, but sent could come in handy if the other station has not received your card, or if you just want to print it on a card with several QSOs.

I see now that I did not add the QSL status to the QSO List table, so I will add that. That could then be used to help you decide if you want to send again or not.

dg9vh commented:
Ok, if the Status is in the results-table, this PR could be dropped :-)

dg9vh commented:
Idea for this: append a notice to the internal notes field to remind the second (third...) sending of a qsl...
the field is "notes" in the edit-mode of QSO... Idea is to keep the existing text and add a new info with timestamp like: "2021-09-26: further QSL-card send by print-qsl-function". Best with empty line as separator

AndreasK79 commented:
Not a bad idea. I'll make a note of it.

As for QSL status. What kind you like.
This?
![image](https://user-images.githubusercontent.com/6977712/134822654-40b5abbe-c792-4602-97cf-164643938b6c.png)
Or this?
![image](https://user-images.githubusercontent.com/6977712/134822658-24d75c5e-e293-4e97-98dd-432241599f73.png)

The information is there, it's just a matter of changing the view to display it in the table.

dg9vh commented:
Well, I think we should show those you posted there... because those are the both most relevant (for Awards for example)... eQSL is also accepted by some but paper and lotw rules the world :-) at least because of the nature that this is about paper qsls at least paper state should be shown :-)

dg9vh commented:
the one above should be the one

dg9vh commented:
![image](https://user-images.githubusercontent.com/13950650/134822876-c63ad8ee-57d2-4116-89a0-605a8f431a0c.png)

AndreasK79 commented:
Ok, I will include all three, and do as the dashboard does. Only display if in use.

dg9vh commented:
Thats also ok :-)

AndreasK79 commented:
@dg9vh I've added it to the master branch. Test it out, and let me know if there are other things that are missing from the QSLPrint.

dg9vh commented:
OK. I check it

dg9vh commented:
Status-table looks good... but notice-field is not filled, correct?

AndreasK79 commented:
I have not done anything about making a note no. I just changed the view. I've added the note to the TODO-list.

dg9vh commented:
OK, I close this PR then now :-) Because main problem is solved :-)

Am Mo., 27. Sept. 2021 um 18:07 Uhr schrieb Andreas Kristiansen <
***@***.***>:

> I have not done anything about making a note no. I just changed the view.
> I've added the note to the TODO-list.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/magicbug/Cloudlog/pull/1202#issuecomment-928026861>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADKN5OUX6LUH5OMCJTZ2ZOTUECJCXANCNFSM5EY5X33Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Amateurfunk ist das, was uns verbindet! Bleiben wir in Kontakt!
Ortsverband Q03, Neunkirchen

dg9vh commented:
problem solved, closed
